galaxy/templates/job-init-db.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "galaxy.fullname" . }}-init-db-{{ randAlphaNum 5 | lower }}
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+spec:
+  ttlSecondsAfterFinished: 10
+  template:
+    metadata:
+      generateName: "{{ include "galaxy.fullname" . }}-init-mounts-"
+      labels:
+        {{- include "galaxy.labels" . | nindent 8 }}
+      annotations:
+        checksum/galaxy_conf: {{ include (print $.Template.BasePath "/configs-galaxy.yaml") . | sha256sum }}
+        checksum/galaxy_rules: {{ include (print $.Template.BasePath "/configmap-galaxy-rules.yaml") . | sha256sum }}
+        checksum/galaxy_extras: {{ include (print $.Template.BasePath "/configmap-extra-files.yaml") . | sha256sum }}
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
+        - name: {{ .Chart.Name }}-init-postgres
+          image: alpine:3.7
+          command: ['sh', '-c', 'chown 101:101 {{ .Values.persistence.mountPath }}; until nc -z -w3 {{ template "galaxy-postgresql.fullname" . }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
```

Comment:
Setting the FsGroup should do the same thing as the chown here right?

Reply:
Not entirely sure, we've always done the chown, so I assumed it was needed, but I haven't tried without. In general, it doesn't add extra startup time, so my guess for now is that there's nothing lost by keeping it